--- matrixgl/configure.ac~	2009-06-25 03:37:31.000000000 +0200
+++ matrixgl/configure.ac	2009-06-25 11:46:58.000000000 +0200
@@ -119,7 +119,8 @@
    for dir in /usr/lib/xscreensaver/config \
       /usr/lib/X11/xscreensaver/config \
       /usr/share/xscreensaver/config \
-      /usr/local/share/xscreensaver/config ; do
+      /usr/local/share/xscreensaver/config \
+      /usr/share/xscreensaver ; do
       test -d $dir && XSS_CONF=$dir && break
    done
    if test -n $XSS_CONF; then
